docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -420,7 +474,9 @@ kubeadm deb/rpm packages and binaries are built for amd64, arm (32-bit), arm64,
 following the [multi-platform
 proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/multi-platform.md).
 
-Currently, only the pod networks [flannel](https://github.com/coreos/flannel) and [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) work on multiple architectures.
+Only some of the network providers offer solutions for all platforms, please consult the list of
```

Comment:
break into 2 sentences: "Only some of the network providers offer solutions for all platforms. Please consult the list of network providers above..."